documentatie/gebruikersdocumentatie/runnen.md

```diff
@@ -0,0 +1,45 @@
+Het bouwen en runnen van het project kan op verschillende manieren met elk verschillende doeleinden.
+
+## Docker compose
+
+Bedoeld voor development, start de backend in watch mode, assets worden geserved door `vite` en maken dus gebruik van Hot Module Reloading:
+
+```sh
+docker compose up
```

Comment:
I get the following error when I run this:
"error while interpolating services.api.user: required variable USER_ID is missing a value: USER_ID not set"

Did I miss some step in setting up docker or docker compose?